Readme.asciidoc

```diff
@@ -1,68 +1,116 @@
-Red Hat JBoss Data Grid Online Services
-=======================================
+= Red Hat JBoss Data Grid Online Services
 
 Red Hat JBoss Data Grid Online Services consists of a multiple services:
 
-- Caching Service - the simplest, default cache
-- Shared Memory Service - a persistent cache
+- Caching Service - In this service, JBoss Data Grid is configured as an in-memory caching service without persistence. Data in the cache will be lost if the service is stopped or restarted, and therefore, it should not be used as a permanent store for data. It can be used to improve your application’s response time, by keeping frequently and recently accessed data in a high speed in-memory store.
```

Comment:
You have references to "JDG" later in the document. In the beginning of the document you should say "In this service, JBoss Data Grid (JDG) is configured..." to explicitly tell the reader JDG means JBoss Data Grid.

Reply:
Done. I modified the first sentence and mentioned in the first paragraph.